kinesis.go

```diff
@@ -131,33 +174,44 @@ func (k *kinesis) setShardIterator(shardIter string) error {
 }
 
 func (k *kinesis) checkActive() (bool, error) {
-	status, err := k.client.DescribeStream(k.args())
+	status, err := k.client.DescribeStream(&awsKinesis.DescribeStreamInput{
+		StreamName:            aws.String("StreamName"), // Required
```

Comment:
Did go-kinesis use a struct for a superset of all args?  The old k.args() had all sorts of crazy stuff in there.